knowledge-base/postgres-stored-functions-in-sqldatasource-wizard.md

```diff
@@ -22,15 +22,14 @@ res_type: kb
 ## Description
 In PostgreSQL prior to version 11 there are Stored functions. Stored Procedures have been introduced with version 11.
 
-1. When using ODBC provider for PostgreSQL with Stored functions, the Stored Procedure drop-down menu of the Report Designer's SqlDataSource wizard is populated, but the returned table columns will be listed as Stored Procedure arguments along with the real arguments, and the Stored function will not be correctly discovered.
-
+1. When using ODBC provider for PostgreSQL with Stored functions, the Stored Procedure drop-down menu of the Report Designer's SqlDataSource wizard is populated, but the returned table columns will be listed as Stored Procedure arguments along with the real arguments, and the Stored function will not be correctly discovered, resulting in an exception.
```

Comment:
"but the returned table columns" is not quite correct - it may be a single value instead of a table (i.e. scalar value instead of a result set). Something like "... but the returned scalar values or result set columns are included in the stored procedure's input arguments..." describes the incorrect behavior better, in my opinion.

Reply:
modified as advised